nodejs/webrtc-hello-again/simple_page.html

```diff
@@ -0,0 +1,166 @@
+<html>
+    <head>
+        <link rel="icon" href="data:;base64,iVBORw0KGgo=">
+        <script src="BandwidthRtc.bundle.js"></script>
+        <script language="javascript">
```

Comment:
I think best practice would be to put this script in the body, or wrap its contents in a callback that fires after the page finishes loading. Otherwise there is a chance (though unlikely in practice) that the elements you're getting by ID may not be there yet when those functions run. I'm not 100% sure if loading the bundle is guaranteed to have finished before this script is executed either, which could lead to SDK initialization issues if it's coming from a CDN etc.